visualization/app/codeCharta/util/customConfigHelper.spec.ts

```diff
@@ -582,9 +582,12 @@ describe("CustomConfigHelper", () => {
 			FileDownloader.downloadData = jest.fn()
 
 			CustomConfigHelper.downloadCustomConfigs(exportedCustomConfigs, CustomConfigFileStateConnector.prototype)
+
+			const timestampFormat = "\\d{4}-\\d{02}-\\d{2}_\\d{2}-\\d{2}"
```

Comment:
I would further restrict the timestamp format 
something like this regexp
`2\\d{3}-(0[1-9]|1[0-2])-(0[1-9]|[12]\\d|3[01])_([0-1]?[0-9]|2[0-3])-[0-5][0-9]`